eng/CHECKENFORCER

```diff
@@ -0,0 +1,2 @@
+format: v0.1-alpha
```

Comment:
Does checkenforcer work for non-master branches?

Reply:
Yes ... it does, but it only reads its configuration from master.